tools/add-feed.js

```diff
@@ -27,10 +27,10 @@ async function add() {
 
   await feedQueue
     .add(feed, {
-      attempts: process.env.FEED_QUEUE_ATTEMPTS || 8,
+      attempts: process.env.FEED_QUEUE_ATTEMPTS || 2,
       backoff: {
         type: 'exponential',
-        delay: process.env.FEED_QUEUE_DELAY_MS || 60 * 1000,
+        delay: process.env.FEED_QUEUE_DELAY_MS || 30 * 1000,
```

Comment:
am I right in thinking this says there's a delay of 30000, or a delay of (30 * 1000) if thats the case, why do we multiply opposed to just write 30000?

Reply:
@Grommers00  I think this is just to provide some context:
`1sec = 1000msecs`
`30 * 1000secs = 30000 msecs = 30 secs`

Sometimes it's good to break down some values to explain where they're coming from. If I do:
`const val = 24 * 60 * 60 * 1000`
it's easy to guess that I'm using `milliseconds in a day`, as opposed to:
`const val = 86400000`
which seems like a random number coming out of nowhere.

Reply:
Makes sense.